Routes/users.js

```diff
@@ -10,10 +10,11 @@ router.get("/", async (req, res) => {
 });
 
 router.post("/", async (req, res) => {
-  const { email } = req.body;
+  const { email, id } = req.body;
```

Comment:
Lets be consistent and label the id being passed through the request. In this case, let's use userID